docs/source/developers/guide/step_by_step/solving_the_problem.rst

```diff
@@ -0,0 +1,28 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+.. _solving:
+
+**********************
+Solving the problem 🧐
```

Comment:
What is "the problem" here? If I see "Solving the problem" in a table of contents, I don't really know what to expect from the target page :-)

Reply:
In general it's about finding yourself around the library (bindings, tests, where to write, how to search ...). Will rephrase if it is unclear ... thinking ...

Reply:
If it's not clear what this page will contain, it can be removed for now (it's empty) and re-added if the need surfaces.